gwvolman/utils.py

```diff
@@ -253,6 +253,11 @@ def _launch_container(volumeName, nodeId, container_config, tale_id='', instance
         )
     host = 'tmp-{}'.format(new_user(12).lower())
 
+    # Add licences mount for STATA and Matlab support
+    mounts.append(
+            docker.types.Mount(type='bind', source="/licenses/", target="/licenses")
+    )
+
```

Comment:
It needs to be umounted in `gwvolman/tasks.py:remove_volume`

Reply:
I've added the umount call to this and the Stata PR https://github.com/whole-tale/gwvolman/pull/120/commits/89b322baf12b9287d0d439ad38f32e15304ace70

Reply:
As discussed, this is not needed